octopusdeploy_framework/util/random.go

```diff
@@ -0,0 +1,47 @@
+package util
```

Comment:
Can we move this to `internal` and use it in both `octopusdeploy` and `octopusdeploy_framework` so we don't have to maintain duplicate code.